libs/accounts/recovery-phone/src/lib/sms.manager.ts

```diff
@@ -29,12 +29,47 @@ export class SmsManager {
   ) {}
 
   public async phoneNumberLookup(phoneNumber: string) {
-    const result = await this.client.lookups.v2
-      .phoneNumbers(phoneNumber)
-      .fetch({ fields: this.config.extraLookupFields.join(',') });
-    // Calling toJSON converts PhoneNumberInstance into a
-    // object that just holds state and can be serialized.
-    return result.toJSON();
+    try {
+      this.metrics.increment('sms.phoneNumberLookup.start');
+      const result = await this.client.lookups.v2
+        .phoneNumbers(phoneNumber)
+        .fetch({ fields: this.config.extraLookupFields.join(',') });
+
+      this.metrics.increment('sms.phoneNumberLookup.success');
+
+      /**
+       * See https://www.twilio.com/docs/lookup/v2-api/sim-swap for more details
+       * on the sim swap object returned by Twilio. We currently track only
+       * the last sim swap period.
+       */
+      const simSwapPeriod = result.simSwap?.lastSimSwap?.swappedPeriod;
+      if (simSwapPeriod) {
```

Comment:
Note that simSwapping seems most useful on confirm code for signin, and we don't call phoneNumberLookup at this here at this point in time.